packages/language-support/src/syntaxColouring/syntaxColouringHelpers.ts

```diff
@@ -148,27 +148,32 @@ export function toParsedTokens(
   });
 }
 
-export function getCypherTokenType(token: Token): CypherTokenType {
-  const tokenNumber = token.type;
-
-  if (tokenNumber === CypherLexer.EOF && token.text !== '<EOF>') {
-    const tokenText = token.text;
-    if (tokenText.startsWith('"') || tokenText.startsWith("'")) {
-      return CypherTokenType.stringLiteral;
-    } else if (tokenText.startsWith('/*')) {
-      return CypherTokenType.comment;
+export function getCypherTokenType(
+  token: Token,
+  nextToken: Token | undefined,
```

Comment:
Potentialyl we could breakout the `isCommentOpener` helper function and use it here to avoid passing `nextToken` and to get more reuse of the comment detection logic?